src/main/java/Duke.java

```diff
@@ -34,17 +36,43 @@ public static void main(String[] args) {
                     System.out.println("____________________________________________________________");
                     System.out.println("List so far: ");
                     System.out.println("Nice! I've marked this task as done:");
-                    System.out.println("[" + t[task_index - 1].getStatus() + "] " + t[task_index - 1].name);
+                    System.out.println(t[task_index - 1]);
                     System.out.println("____________________________________________________________");
                     userInput = in.nextLine();
                 }
                 else {
-                    System.out.println("____________________________________________________________");
-                    System.out.println("added: " + userInput);
-                    System.out.println("____________________________________________________________");
-                    t[listIndex] = new Task(userInput);
-                    listIndex++;
-                    userInput = in.nextLine();
+                    if(userInput.startsWith("todo")) {
+                        t[listIndex] = new Task(userInput.replaceFirst("todo ", ""));
+                        System.out.println("____________________________________________________________");
+                        System.out.println("Got it. I've added this task: ");
+                        System.out.println("[T]" + t[listIndex]);
+                        System.out.println("Now you have " + (listIndex + 1) + " tasks in the list");
+                        System.out.println("____________________________________________________________");
+                        listIndex++;
+                        userInput = in.nextLine();
+                    }
+                    else if(userInput.startsWith("event")){
```

Comment:
Suggestion: 
Use inheritance (A-Inheritance) and polymorphism to override printing commands instead of putting all these code in the main method.